### PR TITLE
DEV-5

### PR DESCRIPTION
**Pull request by MikhailChigir**

### CODE REVIEW

File: task_DEV-5/DEV-5/EntryPoint.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace DEV_5
+{
+    internal class EntryPoint
```
Comment (AlexeiRomankov) [inline task_DEV-5/DEV-5/EntryPoint.cs:6]: please describe the program

File: task_DEV-5/DEV-5/EntryPoint.cs [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    {
+        private static void Main(string[] args)
+        {
+            var list = new List<IFlyable> {new Bird(), new Plane(), new SpaceShip()};
+
+            foreach (var item in list)
+            {
+                item.FlyTo(new Point(100, 200, 800));
```
Comment (AlexeiRomankov) [inline task_DEV-5/DEV-5/EntryPoint.cs:14]: please do not use magic numbers

File: task_DEV-5/DEV-5/Bird.cs [hunk 3]
```diff
@@ -0,0 +1,35 @@
+﻿using System;
+
+namespace DEV_5
+{
+    internal class Bird : IFlyable
```
Comment (AlexeiRomankov) [inline task_DEV-5/DEV-5/Bird.cs:5]: please add summaries

File: task_DEV-5/DEV-5/Bird.cs [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+        public void FlyTo(Point NewPoint)
+        {
+            targetPoint = NewPoint;
+        }
+
+        public double GetFlyTime()
+        {
+            var dist = GetDistance(startPoint, targetPoint);
+            var speed = new Random().Next(0, 20);
+            return dist / speed;
```
Comment (AlexeiRomankov) [inline task_DEV-5/DEV-5/Bird.cs:24]: please add blank line before 'return'

File: task_DEV-5/DEV-5/EntryPoint.cs (hunk 2 above)
Comment (MikhailChigir) [reply task_DEV-5/DEV-5/EntryPoint.cs:14]: Sorry, what do words "magic numbers" mean?

Comment (AlexeiRomankov) [reply task_DEV-5/DEV-5/EntryPoint.cs:14]: Numbers that are not assigned to variables and carry some meaning.
It is better to use  class fields for example minSpeed = 0 and maxSpeed = 20
xCoordinate, yCoordinate ...
startSpeed = 200 etc

Comment (AlexeiRomankov) [reply task_DEV-5/DEV-5/EntryPoint.cs:14]: is it OK for you?

**Pull request merged**